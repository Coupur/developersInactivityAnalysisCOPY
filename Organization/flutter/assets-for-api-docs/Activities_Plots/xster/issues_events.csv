id,date,event,creator_login
1806460352,2018-08-23 22:05:14,review_requested,xster
1806570699,2018-08-23 23:27:34,head_ref_force_pushed,xster
1811735032,2018-08-27 22:26:16,referenced,xster
1811735036,2018-08-27 22:26:16,merged,xster
1811735046,2018-08-27 22:26:16,closed,xster
1811735168,2018-08-27 22:26:21,head_ref_deleted,xster
