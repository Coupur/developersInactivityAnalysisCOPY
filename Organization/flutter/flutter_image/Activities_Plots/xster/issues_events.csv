id,date,event,creator_login
1702720613,2018-06-27 00:01:20,mentioned,xster
1702720614,2018-06-27 00:01:20,subscribed,xster
1532290520,2018-03-21 00:52:14,review_requested,xster
1532440047,2018-03-21 04:24:44,referenced,xster
1532440048,2018-03-21 04:24:44,merged,xster
1532440050,2018-03-21 04:24:44,closed,xster
