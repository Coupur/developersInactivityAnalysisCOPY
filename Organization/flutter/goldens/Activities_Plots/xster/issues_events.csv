id,date,event,creator_login
2298825420,2019-04-25 00:32:42,referenced,xster
2298825422,2019-04-25 00:32:42,merged,xster
2298825425,2019-04-25 00:32:43,closed,xster
2298825459,2019-04-25 00:32:46,head_ref_deleted,xster
2298484664,2019-04-24 20:59:14,mentioned,xster
2298484665,2019-04-24 20:59:14,subscribed,xster
2295672532,2019-04-24 00:05:48,referenced,xster
2295672535,2019-04-24 00:05:48,merged,xster
2295672536,2019-04-24 00:05:48,closed,xster
2295672596,2019-04-24 00:05:51,head_ref_deleted,xster
2153332642,2019-02-20 22:07:29,referenced,xster
2153332643,2019-02-20 22:07:29,merged,xster
2153332645,2019-02-20 22:07:29,closed,xster
2153332746,2019-02-20 22:07:31,head_ref_deleted,xster
